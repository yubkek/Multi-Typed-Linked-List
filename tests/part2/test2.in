INSERT 0 0 WAT
INSERT
NEW 3
1
2
3
INSERT 0 3 4
INSERT 0 6 5
