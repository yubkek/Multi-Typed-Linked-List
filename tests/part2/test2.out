INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: TYPE
List 0: 1 -> 2 -> 3
List 0: 1 -> 2 -> 3 -> 4
INVALID COMMAND: INSERT